micro-airport-location/mongo-statefulset.yml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: mongo-v1
+  namespace: default
+  labels:
+    app: mongo
+    version: v1
+spec:
+  serviceName: mongo
+  replicas: 3
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: mongo
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: mongo
+        version: v1
+    spec:
+      containers:
+        - name: mongo
+          image: mongo:5.0.16
+          imagePullPolicy: IfNotPresent
```

Comment:
@cichan02 should be Always
we expect our pod to be crashed each time the container is not found in the registry